.github/pull_request_template.md

```diff
@@ -0,0 +1,45 @@
+<!--  Thanks for sending a pull request! -->
+
+## Proposed changes
+
+Please include a summary of the changes here and why we need those changes. And also let us know which issue is fixed. 
+
+Fixes # <issue_number_here>
+
+<hr>
+
+## Types of changes
+
+<!-- to mark a point done, place an x in square brackets. eg [x]-->
+<!-- - [x] done with this task-->
+<!----Please delete options that are not relevant. And in order to tick the check box just but x inside them for example [x] like this----->
+
+- [ ] Bug fix 
+- [ ] New feature added
+- [ ] Documentation Update 
+
+<hr>
+
+## Please make sure to follow these points 
+
+<!-- to mark a point done, place an x in square brackets. eg [x]-->
+<!-- - [x] done with this task-->
+<!----Please delete options that are not relevant. And in order to tick the check box just but x inside them for example [x] like this----->
+
+- [ ] I have read the contributing guidelines.
+- [ ] I have performed a self-review of my own code or work.
+- [ ] I have commented my code, particularly in hard-to-understand areas.
+- [ ] My changes generates no new warnings.
+- [ ] My Pull Request title is in format <code>< issue name ></code> eg <code>Added links in the documentation</code>.
+- [ ] I have added tests that prove my fix is effective or that my feature works.
```

Comment:
Can you add a checkbox about code coverage here please

Reply:
added